File: Daily Challanges/2026-04-April/2026-04-13.py
def get_initials(name):
    parts = name.split(' ')
    initials = []
    for part in parts:
        initials.append(part[0].upper() + '.')
    return "".join(initials)

print(get_initials("Tommy Millwood"))
print(get_initials("Savanna Puddlesplash"))
print(get_initials("Dragon"))

File: Daily Challanges/2026-04-April/2026-04-13.js
function getInitials(name) {
  const parts = name.split(" ");
  let initials = "";
  for (let part of parts) {
    initials += part[0].toUpperCase() + ".";
  }
  return initials;
}

console.log(getInitials("Tommy Millwood"));
console.log(getInitials("Savanna Puddlesplash"));
console.log(getInitials("Frances Cowell Conrad"));
